engines/director/score.cpp

```diff
@@ -325,6 +325,7 @@ void Score::update() {
 	if (!debugChannelSet(-1, kDebugFast)) {
 		bool keepWaiting = false;
 
+		//debugC(5, kDebugLoading, "Score::update(): nextFrameTime: %d, time: %d", _nextFrameTime, g_system->getMillis(false));
```

Comment:
Why commenting out? if it is too noisy, put it deeper

Reply:
Fixed.